README.md

```diff
@@ -252,10 +252,11 @@ Theoretical aspects of JaxSim are based on Chapters 7 and 8 of the following Ph.
 
 | Author | Maintainers |
```

Comment:
```suggestion
| Authors | Maintainers |
```

Reply:
Thanks! Done with rebase in https://github.com/ami-iit/jaxsim/pull/282/commits/cc16393942c2747ccb96482cc3f49c611747396f